.github/workflows/go-cd.yml

```diff
@@ -0,0 +1,186 @@
+name: Go - Continuous Deployment
+
+env:
+  TESTING_VERSION: "v0.0.9"
+
+on:
```

Comment:
We use 4 spaces ident in yml files. You can run `prettier` to fix the file for you.
See `.github/workflows/lint-yaml.yml`